src/cosmos/info/coreumInfo.ts

```diff
@@ -1,11 +1,132 @@
 import type { Chain } from '@chain-registry/types'
-import { EdgeCurrencyInfo } from 'edge-core-js/types'
+import { EdgeCurrencyInfo, EdgeTokenMap } from 'edge-core-js/types'
 
 import { makeOuterPlugin } from '../../common/innerPlugin'
+import { makeMetaTokens } from '../../common/tokenHelpers'
 import type { CosmosTools } from '../CosmosTools'
 import type { CosmosNetworkInfo } from '../cosmosTypes'
 import data from '../info/chain-json/coreum.json'
 
+const builtinTokens: EdgeTokenMap = {
+  'usara-core1r9gc0rnxnzpq33u82f44aufgdwvyxv4wyepyck98m9v2pxua6naqr8h03z': {
+    currencyCode: 'SARA',
+    displayName: 'Pulsara',
+    denominations: [{ name: 'SARA', multiplier: '1000000' }],
+    networkLocation: {
+      contractAddress:
+        'usara-core1r9gc0rnxnzpq33u82f44aufgdwvyxv4wyepyck98m9v2pxua6naqr8h03z'
+    }
+  },
+  'ibc/E1E3674A0E4E1EF9C69646F9AF8D9497173821826074622D831BAB73CCB99A2D': {
```

Comment:
Ensure that these denoms actually need to be upper case. If not then use the lowercase for the `tokenId`